app/assets/javascripts/components/edit_campaign_page.js.jsx

```diff
@@ -31,29 +31,12 @@ CampaignPageForm = React.createClass({
     return(
       <div className="form-group">
         <label htmlFor="campaign_page_title" className="control-label">Title</label>
-        <input value={ this.props.title } type="text" id="campaign_page_title" className="form-control" name="campaign_page[title]" ref="title" />
+        <input defaultValue={ this.props.title } type="text" id="campaign_page_title" className="form-control" name="campaign_page[title]" ref="title" />
```

Comment:
according to the react warning, if you use `value` instead of `defaultValue`, then no matter what the user puts in the input, it will submit the original `value`